example/rabbitmq/src/main/java/org/apache/iotdb/rabbitmq/RabbitMQConsumer.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.rabbitmq;
+
+import org.apache.iotdb.rpc.IoTDBConnectionException;
+import org.apache.iotdb.rpc.StatementExecutionException;
+import org.apache.iotdb.session.Session;
+import org.apache.iotdb.tsfile.file.metadata.enums.CompressionType;
+import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
+import org.apache.iotdb.tsfile.file.metadata.enums.TSEncoding;
+
+import com.rabbitmq.client.AMQP;
+import com.rabbitmq.client.BuiltinExchangeType;
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.DefaultConsumer;
+import com.rabbitmq.client.Envelope;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * The class is to show how to get data from RabbitMQ. The data is sent by class RabbitMQProducer.
+ */
+public class RabbitMQConsumer {
+
+  private static final Logger logger = LoggerFactory.getLogger(RabbitMQProducer.class);
+
+  private Session session;
+
+  public RabbitMQConsumer() {
+    try {
+      session =
+          new Session(
+              Constant.IOTDB_CONNECTION_HOST,
+              Constant.IOTDB_CONNECTION_PORT,
+              Constant.IOTDB_CONNECTION_USER,
+              Constant.IOTDB_CONNECTION_PASSWORD);
+      session.open();
+      session.setStorageGroup(Constant.STORAGE_GROUP);
+      for (String[] sql : Constant.CREATE_TIMESERIES) {
+        createTimeseries(sql);
+      }
+    } catch (Exception e) {
+      logger.error(e.getMessage());
+    }
```

Comment:
I think it is better to move setStorageGroup and createTimeseries out of here. 
Call these methods in main.